buildscript/src/main/kotlin/Dependencies.kt

```diff
@@ -54,6 +54,7 @@ object Dependencies {
 
     //https://r8.googlesource.com/r8/
     val r8 = "com.android.tools:r8:1.6.89"
+    val proguardRetrace = "net.sf.proguard:proguard-retrace:6.2.2"
```

Comment:
Does this version correlate with R8 version in a client project?

Reply:
I don't think so. As I understand the format of mapping files as old as ProGuard so there is no need to keep these versions in some kind of sync.